base/data/NGO.js

```diff
@@ -262,6 +264,17 @@ NGO.fetchDonationData = ads => {
 		// HACK: charity objs have conflicting IDs, force NGO to use id instead of @id
 		charity['@id'] = undefined;
 		charity.originalId = charityOriginal.id; // preserve for donation look-up
+
+		// Add local overrides from campaign
+		if (campaign && campaign.localCharities) {
+			if (campaign.localCharities[charity.id]) {
+				Object.keys(campaign.localCharities[charity.id]).forEach(key => {
```

Comment:
Could this merge be done with `Object.assign()`?

Reply:
If Object.assign is used it will overwrite data with empty values - e.g. "" will overwrite "the actual description"